graders/r/r_autograder/pltest.R

```diff
@@ -15,6 +31,7 @@ result <- tryCatch({
 
     ## Directory with test files
     tests_dir <- "/grade/tests"
+    print(list.files())
```

Comment:
Does this belong here? This looks like it might have been left over from testing.